osu.Game/Online/API/Requests/Responses/CommentBundle.cs

```diff
@@ -47,17 +47,27 @@ public List<Comment> Comments
         [JsonProperty(@"included_comments")]
         public List<Comment> IncludedComments { get; set; }
 
+        private List<long> userVotes;
+
         [JsonProperty(@"user_votes")]
-        private List<long> userVotes
+        public List<long> UserVotes
         {
-            set => value.ForEach(v =>
+            get => userVotes;
+            set
             {
-                Comments.ForEach(c =>
+                userVotes = value;
+                userVotes.ForEach(v =>
                 {
-                    if (v == c.Id)
-                        c.IsVoted = true;
+                    foreach (var c in Comments)
```

Comment:
not sure if this can be easily resolved, but this could nullref depending on the order in which `Comments` and `UserVotes` are initialised.

Reply:
So far there were no such cases

Reply:
Obviously. Because you were aware of it. But if the order happened to get changed for whatever reason (ie. the API returned it in a different order) it will crash.

Reply:
How about making something like `CommentBundleController` which will set all the relations between objects?

Reply:
How about binding to `Success` in the `APIRequest` itself?